### PR TITLE
feat: XKCD source plugin UI v2

### PR DESCRIPTION
**Pull request by rhino1998**
#### Summary

Tweak the XKCD source plugin UI
* Add logo
* Change card title to "Configure source"
* Add helper text to source field

### CODE REVIEW

File: plugins/source/xkcd/cloud-config-ui/src/form/index.tsx [hunk 1]
```diff
@@ -49,18 +55,27 @@ export function Form({ initialValues }: Props) {
...
+            <Stack>
               <Controller
                 control={control}
                 name="name"
                 render={({ field, fieldState }) => (
                   <TextField
                     error={!!fieldState.error}
                     fullWidth={true}
-                    helperText={fieldState.error?.message}
+                    helperText={
```
Comment (cduflo) [inline plugins/source/xkcd/cloud-config-ui/src/form/index.tsx:76]: @rhino1998 we have a new pattern (very recent, you didn't miss it) to use the `getFieldHelperText` util from the plugin-config-ui-lib that will keep showing the helper text in addition to the error text

File: plugins/source/xkcd/cloud-config-ui/public/images/xkcd.png
Comment (cduflo) [inline plugins/source/xkcd/cloud-config-ui/public/images/xkcd.png:1]: so minor, but may be worth converting to webp

**Review by cduflo — APPROVED** [review]
nothing blocking, 2 small suggestions
nice!! 🚀

File: plugins/source/xkcd/cloud-config-ui/src/form/index.tsx (hunk 1 above)
Comment (rhino1998) [reply plugins/source/xkcd/cloud-config-ui/src/form/index.tsx]: [a475603](https://github.com/cloudquery/cloudquery/pull/18837/commits/a4756037d7e34c4026289b600425903219e23a58)

File: plugins/source/xkcd/cloud-config-ui/public/images/xkcd.png
Comment (rhino1998) [reply plugins/source/xkcd/cloud-config-ui/public/images/xkcd.png:1]: [9a17aeb](https://github.com/cloudquery/cloudquery/pull/18837/commits/9a17aeb2e23ef1ae5a6704e4fe08247a411ad5bc)

**Pull request merged**